cms/djangoapps/contentstore/tests/test_import.py

```diff
@@ -219,26 +219,6 @@ def test_rewrite_reference_list(self):
             conditional_module.show_tag_list
         )
 
-    def test_rewrite_reference(self):
```

Comment:
This test does the same as the test above `test_rewrite_reference_list`, just with a peergrading sample course.